### PR TITLE
add organization picker

### LINKED ISSUE #205: Organization Picker
A Picker which allows the user to pick the organization, in which they currently want to operate in
Group 33393 (2)

### PR DESCRIPTION
**Pull request by DasProffi**
closes #205
Capture

### CODE REVIEW

File: apps/tasks/lib/pages/login_page.dart [hunk 1]
```diff
@@ -33,9 +34,9 @@ class _LoginPageState extends State<LoginPage> {
           onPressed: () {
             if (_formKey.currentState!.validate()) {
               _formKey.currentState!.save();
-              // TODO do use email password here
+              // TODO do use email password here for login
```
Comment (use-to) [inline apps/tasks/lib/pages/login_page.dart:37]: This should be clear now - maintainer meeting!

File: apps/tasks/lib/pages/organization_picker_dart.dart [hunk 2]
```diff
@@ -0,0 +1,87 @@
+import 'package:flutter/material.dart';
+import 'package:helpwave_localization/localization.dart';
+import 'package:helpwave_theme/constants.dart';
+import 'package:helpwave_widget/loading.dart';
+import 'package:helpwave_widget/widgets.dart';
+import 'package:tasks/pages/home_page.dart';
+
+/// Page for Picking the Organizations the us
```
Comment (use-to) [inline apps/tasks/lib/pages/organization_picker_dart.dart:8]: What?

File: apps/tasks/lib/pages/organization_picker_dart.dart [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+                children: [
+                  Text(context.localization!.myOrganizations, style: titleStyle),
+                  const SizedBox(height: distanceSmall),
+                  ...children,
+                  const SizedBox(height: distanceBig),
+                  Text(context.localization!.askForInvite, style: Theme.of(context).textTheme.titleMedium,),
+                  const SizedBox(height: distanceSmall),
+                  Center(
+                    child: TextButton(
+                      // TODO ad something to indicate reloading e.g. loading Spinner for at least 0.5 sec
```
Comment (use-to) [inline apps/tasks/lib/pages/organization_picker_dart.dart:74]: So do something about it!

File: apps/tasks/lib/pages/register_page.dart [hunk 4]
```diff
@@ -73,10 +73,9 @@ class _RegisterPageState extends State<RegisterPage> {
           onPressed: () {
             if (_formKey.currentState!.validate()) {
               _formKey.currentState!.save();
-              // TODO do use email password here
+              // TODO do use email password here for registration
```
Comment (use-to) [inline apps/tasks/lib/pages/register_page.dart:76]: see above

**Review by use-to — CHANGES_REQUESTED** [review]

File: apps/tasks/lib/pages/login_page.dart (hunk 1 above)
Comment (DasProffi) [reply apps/tasks/lib/pages/login_page.dart:37]: seperate issue #192

File: apps/tasks/lib/pages/register_page.dart (hunk 4 above)
Comment (DasProffi) [reply apps/tasks/lib/pages/register_page.dart:76]: seperate issue #192

**Review by use-to — APPROVED** [review]
There still some conflicts, but this is approved.

**Pull request merged**